apps/silverback-drupal/web/modules/custom/silverback_gatsby_test/src/Plugin/GraphQL/Schema/SilverbackGatsbyTestSchema.php

```diff
@@ -93,6 +93,19 @@ public function getResolverRegistry(): ResolverRegistryInterface {
       $builder->produce('url_path')->map('url', $builder->fromParent()),
     ));
 
+    $addResolver('Webform.path',
+      $builder->compose(
+        $builder->produce('entity_url')->map('entity', $builder->fromParent()),
+        $builder->produce('url_path')->map('url', $builder->fromParent())
+      )
+    );
+    $addResolver('Webform.url',
+      $builder->compose(
+        $builder->produce('entity_url')->map('entity', $builder->fromParent()),
+        $builder->callback(fn (Url $url) => $url->setAbsolute()->toString(TRUE)->getGeneratedUrl())
+      )
```

Comment:
Also a candidate for a dedicated `@url` directive with auto-resolvers.

Reply:
`@produceUrl` 😉